model-engine/model_engine_server/common/dtos/llms/vllm.py

```diff
@@ -34,6 +34,25 @@ class VLLMModelConfig(BaseModel):
         description="Maximum GPU memory utilization for the batch inference. Default to 90%.",
     )
 
+    trust_remote_code: Optional[bool] = Field(
+        default=False,
+        description="Whether to trust remote code from Hugging face hub. This is only applicable to models whose code is not supported natively by the transformers library (e.g. deepseek). Default to False.",
+    )
+
+
+class VLLMEngineAdditionalArgs(BaseModel):
+    """Additional arguments to configure for vLLM that are not direct inputs to the vLLM engine"""
+
+    max_gpu_memory_utilization: Optional[float] = Field(
```

Comment:
this is here for backwards compatibilty for v1 batch completions; will remove if we deprecate v1 or update it to take in the update model config definition